Scripts/Taxon/__init__.py

```diff
@@ -105,21 +98,19 @@ def parseblast(self, fileName):
 
 		for record in self.blast:
 			if record.descriptions:
+				OGGroup = 'OG5_' +  record.alignments[0].hit_def.split('_')[-1].strip()
 
 				if record.alignments[0].hsps[0].expect < self.blastCutOff:
 
 					outfile = open('log_'+ self.code,'a') #****
-					outfile.write('e is ' + str(record.alignments[0].hsps[0].expect) + ' for ' + self.code + ' cot off is ' + str(self.blastCutOff) + '\n')
+					outfile.write('e is ' + str(record.alignments[0].hsps[0].expect) + ' for ' + OGGroup + ' cot off is ' + str(self.blastCutOff) + '\n')
 					outfile.close()
 					self.getgrpp(record,interestList) 					
 
 				else:
-					self.blast.remove(record)
 					outfile = open('log_'+ self.code,'a') #****
-					outfile.write('e is ' + str(record.alignments[0].hsps[0].expect) + ' for ' + self.code + ' below cutoff' + str(self.blastCutOff) + '\n')
+					outfile.write('e is ' + str(record.alignments[0].hsps[0].expect) + ' for ' + OGGroup + ' below cutoff' + str(self.blastCutOff) + '\n')
```

Comment:
This makes much more sense. Now the log indicates the OG and value. This is a log file per taxon, before it just repeated the name of the taxon in every line